example_analysis/2.configure_sbs_process_params.ipynb

```diff
@@ -40,15 +40,14 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #34.    from lib.shared.configuration_utils import random_cmap, CONFIG_FILE_HEADER
</pre><p>Was going to suggest you move this function out. Well done</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheeseman-lab/brieflow/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>